indico/modules/search/client/js/components/SearchBox.jsx

```diff
@@ -7,30 +7,71 @@
 
 import PropTypes from 'prop-types';
 import React, {useState} from 'react';
-import {Form} from 'semantic-ui-react';
+import {Form, Icon, Label, Search} from 'semantic-ui-react';
+import './SearchBox.module.scss';
 
 import {Translate} from 'indico/react/i18n';
 
-export default function SearchBox({onSearch}) {
+const resultRenderer = ({label}, keyword) => (
+  <div styleName="option">
+    <span>
+      <Icon name="search" />
+      {keyword}
+    </span>
+    <Label content={label} size="small" />
+  </div>
+);
+
+export default function SearchBox({onSearch, category}) {
   const [keyword, setKeyword] = useState('');
-  const handleChange = event => {
+  const options = !category.isRoot
+    ? [
+        {value: 'category', title: category.title, label: 'In this category ↵'},
+        {value: 'global', title: 'Global', label: 'All Indico ↵'},
```

Comment:
Needs i18n (I'd keep the icon out of the translatable string of course)

Reply:
As discussed, we found it better to keep the `↵` inside the string for RLT languages.